docs/paper/dev/api/scheduler.md

```diff
@@ -0,0 +1,54 @@
+﻿---
+slug: /dev/scheduler
+---
+
+# Scheduler
+
+The `BukkitScheduler` can be used to schedule your code to be run later or run it repeatedly.
+
+## What is a tick?
+
+Every game runs something called a game loop which essentially executes all the logic of the game over and over,
+a single execution of that loop is called a 'tick'.
+
+In Minecraft's case the amount of ticks per second is 20, meaning that the game loop is executed 20 times per second.
+With some math we can see that 1 tick is equal to 50 milliseconds. A tick taking more than 50ms to execute is the moment
+when your server starts to fall behind on its work and lag.
+
+### Converting between SI units and Minecraft ticks
+
+Every method of the scheduler that takes a delay or period uses ticks as a unit of time.
+
+:::warning
```

Comment:
I don’t think this warrants a warning. Maybe just info

Reply:
IMO time units for async methods would be somewhat expected to be milliseconds (when there's no `TimeUnit` param), however they are not therefore the big red warning